src/app/blog/[id]/page.tsx

```
@@ -17,6 +9,13 @@
 import ShareToSocialLink from "@/components/ShareToSocial";
 import { DevToLink, GithubLink } from "@/components/Social/SocialMedia";
 import { config } from "@/src/common/appconfig";
+import type { PageParams } from "@/src/types/types";
```

ESLint

[error] Expected "@/src/types/types" (internal-type) to come before "@/src/common/appconfig" (internal).

(perfectionist/sort-imports)